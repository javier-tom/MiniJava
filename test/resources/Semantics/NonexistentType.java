class Main {
    public static void main(String[] args) {
        System.out.println(new A().run());
    }
}

class A {
    public int run() {
        return 17;
    }

    public Foo stuff() {
        return 5;
    }

    public int bad(Bad b) {
        return 17;
    }
}
