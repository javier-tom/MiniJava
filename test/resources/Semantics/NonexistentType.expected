class A:
  super: null
  fields:
  method bad:
    params: Bad, 
    returns: int
    locals:
  method run:
    params: 
    returns: int
    locals:
  method stuff:
    params: 
    returns: Foo
    locals:

